src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,3 @@
+$grey-star-color: #f3f3f3;
+$standart-inline-pad: 16px;
```

Comment:
The variable for standard inline padding is correctly defined. Ensure all padding values used in your styles are defined here.